### PR TITLE
[HO] Fix aosp branch and target

### PR DESCRIPTION
**Pull request by retroinspect**
- Changed branch name from aosp-master to aosp-main
- Changed target from aosp_cf_x86_64_phone-userdebug to aosp_cf_x86_64_phone-trunk_staging-userdebug

### CODE REVIEW

**Comment by retroinspect** [comment]
@ser-io @jemoreira @k311093  @ikicha PTAL

File: docker/build-android-with-docker.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ function multiline_helper {
 function get_default_lunch_target {
     local def_lunch_target=""
     case "$(uname -m)" in
-        x86_64|i386|x86|amd64) def_lunch_target="aosp_cf_x86_64_phone-userdebug" ;;
+        x86_64|i386|x86|amd64) def_lunch_target="aosp_cf_x86_64_phone-trunk_staging-userdebug" ;;
         aarch64|arm64) def_lunch_target="aosp_cf_arm64_phone-userdebug" ;;
```
Comment (jemoreira) [inline docker/build-android-with-docker.sh:24]: shouldn't the arm target have been fixed too?

Comment (ikicha) [reply docker/build-android-with-docker.sh:24]: @retroinspect PTAL

Comment (retroinspect) [reply docker/build-android-with-docker.sh:24]: > shouldn't the arm target have been fixed too?

aosp-main seems to not have cuttlefish arm64 target .
What branch should be used for arm64 target?

Comment (retroinspect) [reply docker/build-android-with-docker.sh:24]: @jemoreira PTAL

**Pull request merged**